projects/touch-fan/index.js

```diff
@@ -16,17 +17,18 @@ module.exports = {
     methodology: `We count the ETH on ${fan_contract} and ${touch_contract}`,
     misrepresentedTokens: true,
     era: {
-        tvl: sumTokensExport({ owners: [fan_contract, touch_contract], tokens: [nullAddress] }),
-        staking: sumTokensExport({
-            owners: [lock_tft_address, vester_address, stake_tft_contract_address], tokens: [tft_address],
+        tvl: sumTokensExport({
+            tokensAndOwners: [
+                [nullAddress, fan_contract],
+                [nullAddress, touch_contract],
+                [tft_address, lock_tft_address],
```

Comment:
we cant count own tokens under tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl